playbooks/roles/ocp-metallb-operator/tasks/e2e.yml

```diff
@@ -0,0 +1,1707 @@
+---
```

Comment:
@Shilpa-Gokul This file is too big to review :(
Can you create reusable tasks and checks if needed?

Reply:
Will be addressed as part of the following issue https://github.com/ocp-power-automation/ocp4-playbooks-extras/issues/54